db/migrate/20201217163340_update_enterprise_socia_links.rb

```diff
@@ -1,14 +1,11 @@
 class UpdateEnterpriseSociaLinks < ActiveRecord::Migration
-  def instagram_regex
```

Comment:
I disagree here. Migrations are historical artefacts that should do the same even if the code in the models changes. Otherwise you have to work on migrations when you update models and the logic of the migration may accidentally change. Have a look at our documentation: https://github.com/openfoodfoundation/openfoodnetwork/wiki/Database-migrations

You should add a simple version of the `Enterprise` model here and include all the necessary data like this regex.